css/modals/_dialog.scss

```diff
@@ -79,6 +79,7 @@
 
 .modal-dialog-form {
     color: $modalTextColor;
+    margin-top: 5px !important;
```

Comment:
Yes, otherwise some of the AtlasKit components (don't remember which) was overwriting the styling we set.